xml/containers-orchestration.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+    [
+<!ENTITY % entities SYSTEM "entity-decl.ent">
+%entities;
+]>
+<chapter xml:id="cha-orchestration" xml:lang="en"
+    xmlns="http://docbook.org/ns/docbook" version="5.0"
+    xmlns:xi="http://www.w3.org/2001/XInclude"
+    xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Container Orchestration</title>
+ </info>
+ <sect1 xml:id="sec-pod-deployment-with-podman">
+  <title>Pod Deployment with Podman</title>
+
+  <para>
+   In addition to building and managing images, &podman; makes it possible to
+   work with pods. A pod is a group of one or more containers, with shared
+   storage and network resources. A pod usually encapsulates an application
```

Comment:
Storage is not shared across Pods, but can be done via the concept of volumes. Per default the `cgroup`, `ipc`, `net` ,`uts` namespaces are shared. This can be also modified by the user.


```suggestion
   work with pods. A pod is a group of one or more containers, with shared
   resources like the network interface. A pod usually encapsulates an application
```